exp/operator/api/v1alpha4/bootstrapprovider_types.go

```diff
@@ -50,6 +76,15 @@ type BootstrapProviderList struct {
 	Items           []BootstrapProvider `json:"items"`
 }
 
+func (b *BootstrapProviderList) GetItems() []Provider {
+	providers := []Provider{}
+	for _, provider := range b.Items {
+		providers = append(providers, &provider)
+	}
```

Comment:
You are taking the location of a loop variable here, this will cause issues, I think you need to copy the object before you put it into the list else the list will be full of the same value over and over.

You can see what I mean with this playground: https://play.golang.org/p/JAy1S1N1J5X

Have you got a test at all that will prove that this doesn't copy the same item from the list over and over?
```suggestion
	for _, provider := range b.Items {
	        p := provider
		providers = append(providers, &p)
	}
```
Will need to update this for all the different types